offline/docs_ubuntu_22.04.md

```diff
@@ -0,0 +1,595 @@
+# How to install wire
+
+We have a pipeline in  `wire-server-deploy` producing container images, static
+binaries, ansible playbooks, debian package sources and everything required to
+install Wire.
+
+## Installing docker
+On your machine (we call this the "admin host"), you need to have `docker`
+installed (or any other compatible container runtime really, even though
+instructions may need to be modified). See [how to install
+docker](https://docker.com) for instructions.
+
+On ubuntu 22.04, connected to the internet:
+
+```
+apt install docker.io
+```
+
+Ensure the user you are using for the install has permission to run docker, or add 'sudo' to the docker commands below.
+
+
+## Downloading and extracting the artifact
+Create a fresh workspace to download the artifacts:
+
+```
+$ cd ...  # you pick a good location!
+```
+
+Obtain the latest airgrap artifact for wire-server-deploy. Please contact us to get it for now. We are
+working on publishing a list of airgap artifacts.
+
+Extract the above listed artifacts into your workspace:
+
+```
+$ wget https://s3-eu-west-1.amazonaws.com/public.wire.com/artifacts/wire-server-deploy-static-<HASH>.tgz
+$ tar xvzf wire-server-deploy-static-<HASH>.tgz
+```
+Where the HASH above is the hash of your deployment artifact, given to you by Wire, or acquired by looking at the above build job.
+Extract this tarball.
+
+Make sure that the admin host can `ssh` into all the machines that you want to provision. Our docker container will use the `.ssh` folder and the `ssh-agent` of the user running the scripts.
+
+There's also a docker image containing the tooling inside this repo.
+
+## Making tooling available in your environment.
+
+If you don't intend to develop *on wire-server-deploy itself*, you should source the following shell script.
+```
+source ./bin/offline-env.sh
+```
+
+The shell script will set up a `d` alias. Which runs commands passed to it inside the docker container
+with all the tools needed for doing an offline deploy.
+
+E.g.:
+
+```
+$ d ansible --version
+ansible 2.9.27
+  config file = /wire-server-deploy/ansible/ansible.cfg
+  configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
+  ansible python module location = /nix/store/vmz21km0crjx8j21bhd77vzwkpiiq9w0-python3.9-ansible-2.9.27/lib/python3.9/site-packages/ansible
+  executable location = /nix/store/vmz21km0crjx8j21bhd77vzwkpiiq9w0-python3.9-ansible-2.9.27/bin/ansible
+  python version = 3.9.10 (main, Jan 13 2022, 23:32:03) [GCC 10.3.0]
+
+
+```
+
+## Artifacts provided in the deployment tarball.
+
+The following artifacts are provided:
+
+ - `containers-adminhost/wire-server-deploy-*.tar`
+   A container image containing ansible, helm, and other tools and their
+   dependencies in versions verified to be compatible with the current wire
+   stack. Published to `quay.io/wire/wire-server-deploy` as well, but shipped
+   in the artifacts tarball for convenience.
```

Comment:
Well, it's not only *convenience*: The offline bundle is called that way, because the installation might happen in an environment without access to the internet.